docs/conf.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+import sphinx_bootstrap_theme
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../src'))
+sys.path.insert(0, os.path.abspath('../SunFounder_PiCar-V/remote_control/remote_control/driver'))
+sys.path.insert(0, os.path.abspath('../SunFounder_PiCar/picar'))
+#sys.path.insert(0, os.path.abspath('../SunFounder_PiCar/picar/SunFounder_PCA9685/PCA9685.py'))
```

Comment:
Why is this path commented out? Is it supposed to be there or did you just forget it and leave it in?

Reply:
I might need it later. This is just a WIP so I left it in there for now.